Automattic/pocket-casts-ios
# Playback skipping fix download sizes

Fixes #

This change updates the streaming/download code to set the File Size when it exports correctly

## To test

- Start the app
- Ensure that the FF for `Cache Playing Episode` is active
- Start playing an episode that you never downloaded or played before
- Wait for a while until you see in the logs a message saying: `DownloadManager export session: Finished exporting successfully`
- Now open the episode detail
- Check if the episode size is show correctly
- Tap on the Download option where the size of the file show
- Check that the download is automatic because the file was already downloaded for streamming
- Check that size is still correct.

## Checklist
